File: practicas/3/LopezBaruc/Saludo.java
public class Saludo {

    public static void main(String[] args) {
        if (args.length > 0){
            String nombre = args[0];
            System.out.println("Holaa " + nombre + "!!");
        }else{
            System.out.println("Hola Mundoo!!");
        }
        
    }
}

File: practicas/3/LopezFernando/saludo.c
/*
López Morales Fernando Samuel
Sistemas Operativos | 2026-2
*/
#include <stdio.h>
#include <string.h>
#define MAX_S 50
int main(){
	char cadena[MAX_S];
	printf("Ingresa tu nombre: ");
	fgets(cadena, MAX_S, stdin);
	int tam = -1;
	for(int i = 0; i<MAX_S; i+=1){
		if(cadena[i] == '\0' && tam == -1) tam = i;
	}
	cadena[tam-1] = '\0';
	printf("¡Hola %s!\n", cadena);
	return 0;
}
